src/index.ts

```diff
@@ -1,29 +1,50 @@
 import {Command, flags} from '@oclif/command'
+import { readFileSync, writeFileSync } from 'fs';
+
+
+
+const depExtractor = (file: any, depOrDevDep: string)=>{
+  for(let dep in file.jspm[depOrDevDep]){ // going over devDep or dep in jspm
+    const splitVrsnName = file.jspm[depOrDevDep][dep].split('@');
+    
+    const gitOrNpm = splitVrsnName[0].split(':')
+    const depName = gitOrNpm[0] == 'npm' ? gitOrNpm[1] : dep;
+
+    const depVersion = splitVrsnName[1]; 
+    file[depOrDevDep][depName] = depVersion; //inserting dependency 
+}
+}
 
 class JspmToNpmConvertor extends Command {
   static description = 'describe the command here'
-
   static flags = {
     // add --version flag to show CLI version
     version: flags.version({char: 'v'}),
     help: flags.help({char: 'h'}),
     // flag with a value (-n, --name=VALUE)
-    name: flags.string({char: 'n', description: 'name to print'}),
+    name: flags.string({char: 'n', description: 'name of output file', required: true}),
+    // flag with a value (-i, --input=VALUE)
+    input: flags.string({char: 'i', description: 'name of input file', required: true}),
```

Comment:
@noa-moshe Can we have a default value for this and only overwrite if provided? I feel like it would be too constraining to ask for a name each time